dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/ingest/IngestableDataCheckerTest.java

```diff
@@ -1,170 +1,68 @@
- */
 package edu.harvard.iq.dataverse.ingest;
 
-import org.apache.commons.io.FileUtils;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.io.TempDir;
+import static edu.harvard.iq.dataverse.ingest.IngestableDataChecker.STATA_13_HEADER;
+import static org.assertj.core.api.Assertions.assertThat;
 
-import java.io.File;
-import java.io.FileInputStream;
 import java.io.IOException;
-import java.nio.MappedByteBuffer;
-import java.nio.channels.FileChannel;
-import java.nio.file.Path;
+import java.nio.ByteBuffer;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.assertj.core.api.AbstractStringAssert;
+import org.junit.jupiter.api.Test;
 
-/**
- * @author rmp553
- */
 public class IngestableDataCheckerTest {
 
-    @TempDir
-    Path tempFolder;
-
-    public IngestableDataCheckerTest() {
-    }
-
-    @BeforeAll
-    public static void setUpClass() {
-    }
+    private final IngestableDataChecker instance = new IngestableDataChecker();
 
-    @AfterAll
-    public static void tearDownClass() {
+    private ByteBuffer createBufferContaining(final String fileContents)
+            throws IOException {
 
-    }
+        final ByteBuffer buffer = ByteBuffer.allocate(100);
+        buffer.put(fileContents.getBytes());
+        buffer.rewind();
 
-    @BeforeEach
-    public void setUp() {
+        return buffer;
     }
 
-    @AfterEach
-    public void tearDown() {
+    private AbstractStringAssert<?> assertThatADATA(final String content)
+            throws Exception {
 
+        ByteBuffer buff = createBufferContaining(content);
 
+        return assertThat(this.instance.testDTAformat(buff));
     }
 
+    private AbstractStringAssert<?> assertThatSAV(final String content)
+            throws Exception {
 
-    private File createTempFile(String filename, String fileContents) throws IOException {
-
-        if (filename == null) {
-            return null;
-        }
-        File fh = tempFolder.resolve(filename).toFile();
-        fh.createNewFile();
+        ByteBuffer buff = createBufferContaining(content);
 
-        if (fileContents != null) {
-            FileUtils.writeStringToFile(fh, fileContents);
-        }
-
-        return fh;
+        return assertThat(this.instance.testSAVformat(buff));
     }
 
-    private MappedByteBuffer createTempFileAndGetBuffer(String filename, String fileContents) throws IOException {
-
-        File fh = this.createTempFile(filename, fileContents);
-
-        FileChannel srcChannel = new FileInputStream(fh).getChannel();
-
-        // create a read-only MappedByteBuffer
-        MappedByteBuffer buff = srcChannel.map(FileChannel.MapMode.READ_ONLY, 0, fh.length());
-
-        return buff;
-    }
+    @Test
+    public void testingADATA_returnsMimeType_forProperContent() throws Exception {
```

Comment:
When adding new tests, the convention for test methods is to start with the method being tested as prefix, eg `testDTAformat_returnsMimeType_forProperContent`. Check methods below as well.

Reply:
Fixed